provider/image.go

```diff
@@ -741,9 +744,45 @@ func processLogLine(msg string) (string, error) {
 	return info, nil
 }
 
-func configureDockerClient(configs map[string]string) (*client.Client, error) {
+// When `verify` is set, this function will check that the connection to the docker daemon works.
+// If it doesn't and no host was configured, it will try to connect to the user's docker daemon.
+func configureDockerClient(configs map[string]string, verify bool) (*client.Client, error) {
```

Comment:
Do we need to add an (integration?) test to check the `verify=true` path?

Reply:
The integration tests in `/examples` use the actual provider and therefore `verify=true`, as called from `Configure`.